example/spec/parallel/createReconciliationReport/IngestGranule.MYD13Q1_006.input.payload.json

```diff
@@ -31,11 +31,5 @@
         }
       ]
     }
-  ],
-  "pdr": {
```

Comment:
If you think we should just do this for https://github.com/nasa/cumulus/pull/2311/, that's fine as well, although I think we should leave the pdr cleanup in the helper generally.